code/game/objects/structures/vulture_spotter.dm

```diff
@@ -153,10 +158,13 @@
 		user.pixel_y = 0
 
 /// Handler for when the scope is being attached to the tripod
-/obj/structure/vulture_spotter_tripod/proc/on_scope_attach(mob/user, obj/structure/vulture_spotter_tripod/scope)
+/obj/structure/vulture_spotter_tripod/proc/on_scope_attach(mob/user, obj/item/device/vulture_spotter_scope/scope)
 	if(scope_attached)
 		return
 
+	if(istype(scope, /obj/item/device/vulture_spotter_scope/skillless))
```

Comment:
A var may be faster than a type check but I could be wrong

Reply:
I'm not too worried about the speed of var vs typecheck here considering this won't happen more than maybe a dozen times in a round